WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -570,6 +570,10 @@ private extension OrderListViewController {
     /// Removes the selected state otherwise.
     ///
     func highlightSelectedRowIfNeeded(shouldScrollIfNeeded: Bool = false) {
+        guard supportsHighlighting() else {
```

Comment:
Great work finding the cause of this bug @staskus 

I am adding a quick warning to check that that guard doesn't prevent any necessary action from happening. As you can see, that function(`highlightSelectedRowIfNeeded`) contains logic not strictly related to highlight, such as deselecting or scrolling. Would it be ok if we don't execute these actions for iPad OS 16 and 16.1?
I remember that this [PR](https://github.com/woocommerce/woocommerce-ios/pull/12053) addressed a regression caused by a similar change :)

Reply:
Thanks, this is a good comment!

Yes, I haven't found out which part exactly causes the issue since I spent only a few minutes on this PR. However, I can see if adding `highlightSelectedRowIfNeeded` only on `selectRow` part would solve the issue 👍